test/commands/validate.test.ts

```diff
@@ -7,6 +7,9 @@ describe('architect validate component', function () {
   // set to true while working on tests for easier debugging; otherwise oclif/test eats the stdout/stderr
   const print = false;
 
+  const cr = new RegExp(/\r/, 'g');
```

Comment:
What are these used for?

Reply:
oops, these were leftover from previous invalid tests. removed both regex constants.